gizmo.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python3
+import numpy as np
+import pandas as pd
+
+def hello(name, country='Finland'):
+    
+    print('Hello {}, how are things in {}?'.format(name,country))
+    
+    return
+
+
+def spell(str_in):
+    
+    str_out = str_in[0]
+    for el in str_in[1:]:
+        str_out += '.' + el
+    print(str_out)
+    
+    return
+
+
+def relative_path(subject_identifiers):
+    
+    rel_path = './subjects/mock_recording_'
+    ext = '.rec'
+    
+    file_list = [rel_path + elem + ext for elem in subject_identifiers]
```

Comment:
take a look at [f-strings](https://realpython.com/python-f-strings/):

```python
file_list = [f'./subjects/mock_recording_{elem}.rec' for elem in subject_identifiers]
```

Reply:
Thanks for the advice! I was not familiar with this type of string.